backend/api/views.py

```diff
@@ -48,7 +48,7 @@ def eligibility_check(user, data):
         next_step = reverse("api-auditee-info")
 
         # Store step 0 data in profile, overwriting any pre-existing.
-        user.profile.entry_form_data = data
+        user.profile.entry_form_data = serializer.data
```

Comment:
we should store the serialized data instead of the raw data, as the serialized data will have done the appropriate type conversions (e.g. `"True" --> True`)